### ISSUE TITLE
accessibility issues

### ISSUE THREAD

**Issue by tyeung2017** [facn2/ApolloXI#65]
Below are some Accessibility issues after running chrome plugin to check:

[Severe] ARIA attributes which refer to other elements by ID should refer to elements which exist in the DOM (5)
[Severe] ARIA state and property values must be valid (5)
[Severe] Controls and media elements should have labels (10)
[Severe] Elements with ARIA roles must ensure required owned elements are present (5)
[Warning] Meaningful images should not be used in element backgrounds (15)
[Warning] Text elements should have a reasonable contrast ratio (1)
[Warning] The web page should have a title that describes topic or purpose (2)
[Warning] The web page should have the content's human language indicated in the markup (2)
[Warning] Video elements should use <track> elements to provide captions (5)
